### ISSUE TITLE
i need some help,please

### ISSUE THREAD

**Issue by kavinwow100** [Ildaron/Laser_control#4]
I saw this project a few days ago and felt great. I wanted to try to run it. I found "yolov4_tiny_weights_voc.h5" in the project folder. Is this model file well trained? Can it be used to identify mosquitoes and cockroaches? Can you share all the execution code? Thank you.

**Comment by Ildaron** [comment]
@kavinwow100 "yolov4_tiny_weights_voc.h5" in my case was trained only for test in home conditions, so you need to train yourself

**Comment by kavinwow100** [comment]
Thx so much.Do you have some advices for the camera?imx 219 160 degree or any others?

**Comment by Ildaron** [comment]
@kavinwow100 yes, good, if you will need stereovision can consider this company - https://stereopi.com/

**Comment by kavinwow100** [comment]
OK, thank you. I'm still a little worried because mosquitoes are too small to be recognized by the camera. I'm going to buy an imx219 camera with 8million pixels and 160 degrees to try
